test/libs/047-entity-collector.coffee

```diff
@@ -0,0 +1,37 @@
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+
+createHuman = (label)->
+  _id: '91760e03c9c367c3fd75655532f4b2f6'
+  type: "human"
+  labels: { en: label },
+  claims: { 'wdt:P31': [ 'wd:Q5' ] }
+
+checkEntity = __.require 'controllers', 'tasks/lib/check_entity'
```

Comment:
as `checkEntity` is async and non-deterministic (relies on the Wikidata API), those tests should probably be moved to `api_tests`: tests in the `test` folder are run before every commit (see [pre-commit githooks](https://github.com/inventaire/inventaire/blob/master/.githooks/pre-commit/pre-commit#L10)), they should thus run really fast and should fail only in the case something in the code broke the code (not in case of network error or anything of this kind)